### ISSUE TITLE
Winetricks' comctl32 v5 causes crashes in latest Wine releases

### ISSUE THREAD

**Issue by kevgrig** [Winetricks/winetricks#824]
Please see [https://bugs.winehq.org/show_bug.cgi?id=43486](https://bugs.winehq.org/show_bug.cgi?id=43486) and [comment 9](https://bugs.winehq.org/show_bug.cgi?id=43486#c9) from `shell32` maintainer Nikolay Sivov:

> In current wine shell32 depends on comctl32 v6 exports/imagelist behavior. Winetricks install v5 module, and that breaks shell32. [...]

**Comment by austin987** [comment]
Yeah, that's not something I can do much about. Native DLLs can help or hurt.

Do you have a suggestion on how winetricks could improve?

**Comment by kevgrig** [comment]
I wonder if comctl32 v6 might help? I'm not sure where to find that.

**Comment by austin987** [comment]
It's in win7sp1. It may be a bit tricky though because of the manifest requirements. It would be more helpful if I could reproduce it.

I looked at the bug, I have some comments from your original report:
(In reply to averageradical from comment #0)

> I installed Winamp with:
> 
> $ wget
> https://raw.githubusercontent.com/Winetricks/winetricks/master/src/winetricks
> $ chmod +x winetricks 

There's no need to chmod it if you're invoking it with sh, fyi.
> $ WINEARCH='win32' sh winetricks comctl32 wininet ie6 allfonts allfonts wmp9
> d3dx9_36

FYI, you have allfonts listed twice.

> $ wget http://winampplugins.co.uk/Winamp/winamp5666_full_en-us_redux.exe

That redirects to https://getwinamp.com/, which is a webpage saying that it was removed. There's a copy at http://www.meggamusic.co.uk/winamp/, sha256 is 5158d1e92fe476227e9bc739cd15fff4e9e847d8598613a6ae220ce085e3a6fb. Is that correct? Could you upload a screenshot with the menu you're clicking pointed out (I never used winamp much myself).

**Comment by kevgrig** [comment]
Thanks for taking a look! Since that bug report, I found that winetricks itself can install winamp, so now I've been using the following instructions:

    WINEARCH=win32 ./winetricks -q comctl32 directmusic directplay directx9 ie8 mfc42 wmp10 windowmanagerdecorated=n && WINEARCH=win32 ./winetricks winamp

Then on the first execution of Winamp, choose the "Classic Skin". When Winamp launches, there should be a playlist editor at the bottom. Click on the "Add" button in the bottom left and then click on "Add File":

![winamp_add_file](https://user-images.githubusercontent.com/6432587/29050800-39dc8a8a-7b93-11e7-8e74-202a96b0e542.png)

> There's no need to chmod it if you're invoking it with sh, fyi.

Good point, thanks.

> FYI, you have allfonts listed twice.

Yeah, I noticed that too after I submitted! Embarrassing. What is it that causes proofreading to only succeed after I've submitted something for all eternity?!

> I never used winamp much myself

You are missing out, my friend. It's the only reason I use Wine. A few tips: Click Alt+L to open the Media Library and add files there, and also double click on the Equalizer in the main play window (under the time display) to open the majestic Milk Drop visualization engine. Note that I make the following additional customizations to Winamp:

- Right click Winamp > Options > Preferences... > General Preferences > Playlist > select Use font: MS Sans Serif to fix any strange font display issues
- To fix bug [12060](https://bugs.winehq.org/show_bug.cgi?id=12060): Winamp > Options > Preferences... > Plug-ins > Visualization > MilkDrop v2.25c > Configure > WINDOWED settings > Uncheck Integrate with winamp skin
